src/frontend/src/pages/TeamsPage/TeamMembersPageBlock.tsx

```diff
@@ -30,22 +31,34 @@ const TeamMembersPageBlock: React.FC<TeamMembersPageBlockProps> = ({ team }) =>
   const auth = useAuth();
   const [isEditingMembers, setIsEditingMembers] = useState(false);
   const [members, setMembers] = useState(team.members.map(userToAutocompleteOption));
+  const [head, setHead] = useState({
+    id: `${userToAutocompleteOption(team.head).id}`,
+    label: userToAutocompleteOption(team.head).label
+  });
 
-  const { isLoading, isError, error, data: users } = useAllUsers();
+  const { isLoading: allUsersIsLoading, isError: allUsersIsError, error: allUsersError, data: users } = useAllUsers();
   const {
-    isLoading: setTeamMemberIsLoading,
-    isError: setTeamMemberIsError,
-    error: setTeamMemberError,
-    mutateAsync
+    isLoading: setTeamMembersIsLoading,
+    isError: setTeamMembersIsError,
+    error: setTeamMembersError,
+    mutateAsync: setTeamMembersMutateAsync
   } = useSetTeamMembers(team.teamId);
+  const {
+    isLoading: setTeamHeadIsLoading,
+    isError: setTeamHeadIsError,
+    error: setTeamHeadError,
+    mutateAsync: setTeamHeadMutateAsync
+  } = useSetTeamHead(team.teamId);
 
-  if (isError) return <ErrorPage message={error?.message} />;
-  if (setTeamMemberIsError) return <ErrorPage message={setTeamMemberError?.message} />;
-  if (isLoading || setTeamMemberIsLoading || !users) return <LoadingIndicator />;
+  if (allUsersIsError) return <ErrorPage message={allUsersError?.message} />;
+  if (setTeamMembersIsError) return <ErrorPage message={setTeamMembersError?.message} />;
+  if (setTeamHeadIsError) return <ErrorPage message={setTeamHeadError?.message} />;
+  if (allUsersIsLoading || setTeamMembersIsLoading || setTeamHeadIsLoading || !users) return <LoadingIndicator />;
 
   const handleSubmit = async () => {
     try {
-      await mutateAsync(members.map((member) => member.id));
+      await setTeamMembersMutateAsync(members.map((member) => member.id));
+      await setTeamHeadMutateAsync(+head.id);
```

Comment:
why's there a plus here

Reply:
To convert the id from string to number

Reply:
Good point. I suggest we use a parseInt() to convert the id from string to number b/c we're more familiar with it and easier to understand the code.